variables.tf

```diff
@@ -227,14 +227,6 @@ variable "location" {
   description = "Location of cluster, if not defined it will be read from the resource-group"
   default     = null
 }
-
-variable "log_analytics_solution_id" {
```

Comment:
Please preserve this variable, or it'll be considered as a breaking change.

Reply:
Sure thing. I'll add a `tflint-ignore: terraform_unused_declarations` before it so that the linter behaves.

Reply:
If we preserver the reference to this variable in `azurerm_log_analytics_solution`'s `count` expression, we won't need this tflint annotation, right?